Makefile

```diff
@@ -22,7 +22,7 @@ composer:
 	docker-compose run --rm cli composer install
 
 composerupdate:
-	docker-compose run --rm cli bash -c "composer update && composer show -D > versions.json"
+	docker-compose run --rm cli bash -c "composer update && composer show -D --format=json > versions.json"
```

Comment:
Curious why the format change. Granted, the filename indicated it should have been json...oops.